core/src/main/java/org/apache/calcite/prepare/CalciteSqlValidator.java

```diff
@@ -22,10 +22,12 @@
 import org.apache.calcite.sql.SqlOperatorTable;
 import org.apache.calcite.sql.validate.SqlValidatorImpl;
 
-/** Validator. */
-class CalciteSqlValidator extends SqlValidatorImpl {
+/** Validator.
+ *  Public access-level to allow overriding and custom validation logic.
+ * */
```

Comment:
Similar comments to previous javadoc. Second sentence should be a paragraph. Also a sentence.